server-common/src/main/java/org/apache/cassandra/sidecar/common/server/StorageOperations.java

```diff
@@ -102,4 +102,6 @@ default void outOfRangeDataCleanup(@NotNull String keyspace, @NotNull String tab
     {
         outOfRangeDataCleanup(keyspace, table, 1);
     }
+
+    String getOperationMode();
```

Comment:
should be `operationMode` and there are no javadocs here
```suggestion
    String operationMode();
```